coreservices/partsrelationshipservice/prs-api/src/main/java/net/catenax/prs/mappers/PartRelationshipEntityListToDtoMapper.java

```diff
@@ -65,9 +65,14 @@
     public PartRelationshipsWithInfos toPartRelationshipsWithInfos(final Collection<PartRelationshipEntity> source, final Collection<PartIdEntityPart> allPartIds, final Collection<PartAttributeEntity> typeNames, final Collection<PartAspectEntity> aspects) {
```

Comment:
If we do this changes here then can we also this update parameter name ? 
```suggestion
    public PartRelationshipsWithInfos toPartRelationshipsWithInfos(final Collection<PartRelationshipEntity> source, final Collection<PartIdEntityPart> allPartIds, final Collection<PartAttributeEntity> attributes, final Collection<PartAspectEntity> aspects) {
```